ecs-agent/wsclient/client_test.go

```diff
@@ -42,7 +44,10 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-const dockerEndpoint = "/var/run/docker.sock"
+const (
+	dockerEndpoint                  = "/var/run/docker.sock"
```

Comment:
This is something which was preexisting actually. But sure, let me take a look up if it impacts and if it can be something random. Since, this is not blocking, I would pick it up in a later CR, if thats okay.